rt_gene/scripts/estimate_gaze.py

```diff
@@ -44,6 +43,16 @@ def __init__(self, device_id_gaze, model_files):
 
         self.tf_prefix = rospy.get_param("~tf_prefix", "gaze")
         self.headpose_frame = self.tf_prefix + "/head_pose_estimated"
+        self.gaze_backend = rospy.get_param("~gaze_backend", "tensorflow")
+
+        if self.gaze_backend == "tensorflow":
```

Comment:
I now realise that there is something odd going on with the classes. The `GazeEstimator` classes in `estimate_gaze_tensorflow` and `estimate_gaze_pytorch` derive from `GazeEstimatorBase`, and in addition `GazeEstimatorROS` derives from `GazeEstimatorBase`. However, within `GazeEstimatorROS` the `_gaze_estimator` attribute contains an object derived from `GazeEstimatorBase` as well. I think this is confusing.

I suggest to:
1) Derive `GazeEstimatorROS` from `object` instead
2) Rename `estimate_gaze_tensorflow.GazeEstimator` to `GazeEstimatorTensorflow` and similarly for PyTorch.

What do you think @ngageorange?

Reply:
Yes agreed - the initial thought that ros and the standalone version would inherit from the same abstract class but its actually better if the pytorch/tensorflow version do that, hence the change but it seems that I forgot to get the ros version to re-inherit from object!
Thanks for the catch.